src/web/mjs/connectors/OnePieceTube.mjs

```diff
@@ -0,0 +1,48 @@
+import Connector from "../engine/Connector.mjs";
+
+export default class OnePieceTube extends Connector {
+    constructor() {
+        super();
+        super.id = 'onepiecetube';
+        super.label = 'One Piece Tube';
+        this.tags = ['manga', 'german'];
+        this.url = 'http://onepiece-tube.com/';
+    }
+
+    async _getMangas() {
+        return [
+            {
+                id: 'onepiece',
+                title: 'One Piece'
+            }
+        ];
+    }
+
+    // eslint-disable-next-line no-unused-vars
+    async _getChapters(manga) {
+        let request = new Request(this.url + 'kapitel-mangaliste#oben', this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.sagatable table.list tbody tr');
+
+        return data.filter(e => e.rowIndex !== 0).map(element => {
+            let number = element.cells[0].innerText;
+            let title = element.cells[1].innerText;
+            return {
+                id: `kapitel/${number}`,
+                title: `${number} - ${title}`
+            };
+        });
+    }
+
+    async _getPages(chapter) {
+        let request = new Request(`${this.url}${chapter.id}/1`, this.requestOptions);
+        let data = await this.fetchDOM(request, 'td#tablecontrols a');
+
+        data = await Promise.all(data.map(d => {
```

Comment:
Do not start scraping all pages, this will slow down HakuNeko. Use the protocol mechanism(`createConnectorURI`) by returning the HTML links and extract the images in `_handleConnectorURI` like done in e.g. [TAAD](https://github.com/manga-download/hakuneko/blob/c81e82ae03f94e0cd64e760ee3ffcda3bb519cd8/src/web/mjs/connectors/TAADD.mjs#L75-L85)

Reply:
think i fixed it